src/custom/utils/operator/error.ts

```diff
@@ -57,9 +56,11 @@ export default class OperatorError extends Error {
       return OperatorError.apiErrorDetails.UNHANDLED_ERROR
     }
   }
-  static async getErrorForUnsuccessfulPostOrder(response: Response) {
+
+  static async getErrorFromStatusCode(response: Response) {
     switch (response.status) {
       case 400:
+      case 404:
```

Comment:
Will this work? Afaik `NotFound` is not a valid error in the backend, so it won't be handled...

Reply:
it does work, though it's more a workaround until the proper error is returned. you can test it by trying what i wrote in #747 or just going here: https://protocol-rinkeby.dev.gnosisdev.com/api/v1/markets/0x5592E2C0cf4dbc12D3aB100b257153436a1f0FEa-0x6D7F0754FFeb405d23C51CE938289d4835bE3b14/buy/14882129914429537408